pkg/fission-cli/cmd/spec/spec.go

```diff
@@ -197,6 +197,73 @@ func SpecSave(resource interface{}, specFile string) error {
 	return nil
 }
 
+// save saves object encoded value to spec file under given spec directory
+func saveUpdate(data []byte, specDir string, specFile string) error {
```

Comment:
Can we merge the save and saveUpdate functions? The only difference I notice is truncating. We can probably add a flag to the existing function lets `truncate bool` and consider saving the spec accordingly.